src/mahoji/lib/abstracted_commands/slayerTaskCommand.ts

```diff
@@ -202,7 +202,6 @@ async function returnSuccess(channelID: string, user: MUser, content: string) {
 					interaction: selection,
 					...options
 				});
-				return;
```

Comment:
still unsure why slayer is touched by this pr

Reply:
not sure either, fixed.